docs/source/reference-core.rst

```diff
@@ -867,9 +867,9 @@ Nursery objects provide the following interface:
    .. method:: start(async_fn, *args, name=None)
       :async:
 
-      Like :meth:`start_soon`, but blocks until the new task has
-      finished initializing itself, and optionally returns some
-      information from it.
+      Like :meth:`start_soon`, but blocks until the new child task
+      signals that it may unblock, optionally returning additional
```

Comment:
"signals" is a slightly overloaded word to use here due to Unix signals. Maybe "until the new child task indicates that it's finished starting up"? And I think the original "and optionally returns some information" is better than "optionally returning additional information" -- the more active phrasing tends to communicate more clearly IMO.

Reply:
changing to "indicates"